app/assets/javascripts/file_dropzone.js.es6

```diff
@@ -0,0 +1,73 @@
+window.App.FileDropzone = class FileDropzone {
+  constructor(props) {
+    this.props = props
+
+    let file_input_query = document.querySelectorAll("input[type='file']")
+    if (file_input_query.length == 1) {
+      this.insert_dropzone(file_input_query[0])
+    }
+  }
```

Comment:
In the interest of writing less code, and since you already have jquery here, I'd be inclined to write this as:

```
if ($("input[type='file']").length == 1) {
  this.insert_dropzone(file_input_query[0])
}
```